relbench/tasks/amazon.py

```diff
@@ -201,7 +201,7 @@ class ProductLTVTask(RelBenchNodeTask):
     entity_col = "product_id"
     entity_table = "product"
     time_col = "timestamp"
-    target_col = "count_"  # "ltv"
+    target_col = "ltv"
```

Comment:
shall we remove `count_` from sql below?

Reply:
good point - done :)